### PR TITLE
add entities for payload storage in sql db

### PR DESCRIPTION
**Pull request by jon-shipley**
Add mtc_results schema to hold the pupil response data

### CODE REVIEW

File: admin/data/sql/migrations/20200904113433.do.create-user-input-type-table.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+DROP TABLE IF EXISTS mtc_results.userInputType;
+
+CREATE TABLE mtc_results.userInputType
+([id]        INT IDENTITY ( 1, 1 ) NOT NULL,
+ [createdAt] DATETIMEOFFSET(3)     NOT NULL DEFAULT GETUTCDATE(),
+ [updatedAt] DATETIMEOFFSET(3)     NOT NULL DEFAULT GETUTCDATE(),
+ [version]   ROWVERSION,
+ [name]      VARCHAR(10)           NOT NULL,
```
Comment (GuyHarwood) [inline admin/data/sql/migrations/20200904113433.do.create-user-input-type-table.sql:8]: please use nvarchar (as per existing tables) unless you have a specific reason for it.

**Review by GuyHarwood — COMMENTED** [review]
LGSF, just one comment

File: admin/data/sql/migrations/20200904113433.do.create-user-input-type-table.sql (hunk 1 above)
Comment (jon-shipley) [reply admin/data/sql/migrations/20200904113433.do.create-user-input-type-table.sql]: Changed to NVARCHAR for consistency with other tables.

File: admin/data/sql/migrations/20200910145708.do.create-event-table.sql [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+ updatedAt        DATETIMEOFFSET(3)  NOT NULL DEFAULT GETUTCDATE(),
+ version          ROWVERSION,
+ eventType        NVARCHAR(255)      NOT NULL,
+ eventDescription NVARCHAR(4000),
+ CONSTRAINT PK_eventTypeLookup PRIMARY KEY CLUSTERED ([id] ASC) WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON),
+ CONSTRAINT eventTypeLookup_eventType_uindex UNIQUE (eventType)
+);
+
+
+INSERT INTO mtc_results.eventTypeLookup (eventType, eventDescription)
```
Comment (GuyHarwood) [inline admin/data/sql/migrations/20200910145708.do.create-event-table.sql:15]: 💯

**Review by GuyHarwood — APPROVED** [review]
LGTM

**Pull request merged**